docker/r3.6.1/fix-permissions-100.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# set permissions on a directory
+# after any installation, if a directory needs to be (human) user-writable,
+# run this script on it.
+# It will make everything in the directory owned by the group $NB_GID
+# and writable by that group.
+# Deployments that want to set a specific user id can preserve permissions
+# by adding the `--group-add users` line to `docker run`.
+
+# uses find to avoid touching files that already have the right permissions,
+# which would cause massive image explosion
+
+# right permissions are:
+# group=$NB_GID
+# AND permissions include group rwX (directory-execute)
+# AND directories have setuid,setgid bits set
+
+set -e
+
+for d in "$@"; do
+  find "$d" \
+    ! \( \
+      -group 100 \
+      -a -perm -g+rwX  \
+    \) \
+    -exec chgrp 100 {} \; \
```

Comment:
Maybe make this use the `NB_GROUP` environment variable (or is it`NB_GID`)? instead of having two different versions of the same script.

Reply:
NB_GID, and from all my testing, if you set this to 100 to fix the conda directories, then NB_USER can't the conda packages, so it has to be 1000. But, if you set them to 1000, then the user who's running in the notebook doesn't have access, it needs them to be 100.
🤷‍♀